systemd/ua-license-check.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Ubuntu Advantage check for GCP Ubuntu Pro licenses
+After=network.target network-online.target systemd-networkd.service ua-auto-attach.service
```

Comment:
Why do we need to specify all of these extra targets here ? Wouldn't waiting for `ua-auto-attach.service`  enough to run this service ?

Reply:
@lucasmoura auto-attach service is only present on PRO images, in the absence of that service or target, we need to fallback to the next earliest time in boot we need to wait for.... which is that network is "up" on the system so we can talk to the IMDS.

Reply:
Ohh excelent point.